packages/scene-composer/src/components/panels/__tests__/__snapshots__/SceneNodeInspectorPanel.spec.tsx.snap

```diff
@@ -160,7 +160,7 @@ exports[`SceneNodeInspectorPanel returns expected elements. SceneNode panel cont
           >
             <div
               data-mocked="Input"
-              data-test-id="node-ref"
+              data-testid="ip-undefined"
```

Comment:
bad snapshot, should make sure the undefined is gone.

Reply:
https://github.com/awslabs/iot-app-kit/pull/2579